tests/test_openai.py

```
@@ -0,0 +1,204 @@
+        
+        async def mock_reply_text(text, parse_mode=None):
+            nonlocal response_captured
+            response_captured = text
+            logger.info(f"✅ CAPTURED RESPONSE: {text}")
+            return True
+        
+        # Mock the reply_text method on the message object
+        message.reply_text = mock_reply_text
+        
+        # Also mock the _send_telegram_message method to prevent it from trying to send via Telegram
+        original_send_telegram = bot.proactive_agent._send_telegram_message
```

Ruff (0.12.2)

Local variable `original_send_telegram` is assigned to but never used

Remove assignment to unused variable `original_send_telegram`

(F841)